core/src/style/style.cpp

```diff
@@ -95,13 +95,20 @@ void Style::setupShaderUniforms(int _textureUnit, Scene& _scene) {
 
 void Style::onBeginDrawFrame(const View& _view, Scene& _scene, int _textureUnit) {
 
+    // TODO cache which uniforms are used by the shader!
```

Comment:
I see, I saw that coming up in the profiler too at some point. We could declare all the string uniform like `u_time` etc statically in some files to address the string allocations at first and see how it goes, it would reduce the allocations.